dedb.go

```diff
@@ -103,3 +103,49 @@ SELECT t.id::text,
 	log.Debugf("Tags query: %s", query)
 	return tx.tx.QueryContext(ctx, query)
 }
+
+// GetAVUs returns a sql.Rows for CyVerse metadata AVUs with an optional UUID prefix for the ultimate target ID (but still including nested AVUs)
+func (tx *DEDBTx) GetAVUs(ctx context.Context, rootTargetIdPrefix string) (*sql.Rows, error) {
```

Comment:
It appears that this query returns attributes for which the name, value, and unit are all empty. That may not cause a problem, but I thought I'd mention it in case it does.